radicle/src/profile.rs

```diff
@@ -242,14 +243,18 @@ mod test {
         fs::create_dir_all(path.clone()).unwrap();
 
         let last = tmp.path().components().last().unwrap();
-        let home = Home::new(
+        let mut home = Home::new(
             tmp.path()
                 .join("..")
                 .join(last)
                 .join("Home")
                 .join("Radicle"),
         )
         .unwrap();
+        if cfg!(target_os = "macos") {
+            home.path = Path::new("/").join(home.path.strip_prefix("/private").unwrap())
```

Comment:
I'd say use `unrwap_or` just in case the `/private` doesn't show up
```suggestion
            home.path = Path::new("/").join(home.path.strip_prefix("/private").unwrap_or(home.path))
```

Reply:
Note that the join won't add a second `/` https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=fa1c7b98ace3cf2f2174df02472163b8

Reply:
> I'd say use `unrwap_or` just in case the `/private` doesn't show up
> 
> ```suggestion
> 
>             home.path = Path::new("/").join(home.path.strip_prefix("/private").unwrap_or(home.path))
> 
> ```

Good point 👍

Reply:
> Note that the join won't add a second `/` https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=fa1c7b98ace3cf2f2174df02472163b8

Yeah the join is due to `strip_prefix` removing the first `/` so we add it back
see https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=ad6cb1f45ca68d20f88fd5ef0f365748

Reply:
> Sure, what I meant was that if the path already starts with `/` in the case of using `unwrap_or` we're not adding an extra one :)

Ah nice good to know